src/launcher_args.js

```diff
@@ -1,15 +1,23 @@
 const fs = require('fs');
 const argv = require('yargs') // eslint-disable-line
-	.option('config', {
-		alias: 'c',
-		type: 'string',
-		description: 'Path to config.json'
-	})
-	.option('write-path', {
-		alias: 'w',
-		type: 'string',
-		description: 'Path to Spring'
-	}).argv;
+	.command('$0', 'default command', (yargs) => {
+		yargs
+			.option('config', {
+				alias: 'c',
+				type: 'string',
+				description: 'Path to config.json'
+			})
+			.option('write-path', {
+				alias: 'w',
+				type: 'string',
+				description: 'Path to Spring'
+			})
+			.positional('replay_file', {
```

Comment:
should be `replay-file` for consistency with `write-path` (I also find CLI arguments often specified via dash instead of underscore)